### DISCUSSION TITLE
Error while trying to using the model

### DISCUSSION THREAD

**Discussion by mariomagdysamy** [antonior92/automatic-ecg-diagnosis#20] [General]
I had tried to train the model and it's give me this error 
![image](https://user-images.githubusercontent.com/63364182/159624333-8202319f-d2b4-4da3-8443-03b93c841f16.png)
`usage: train.py [-h] [--val_split VAL_SPLIT] [--dataset_name DATASET_NAME]
                path_to_hdf5 path_to_csv
train.py: error: unrecognized arguments: /content/gdrive/MyDrive/Colab Notebooks/gp/data/annotations/dnn.csv`

my command line is "!python train.py /content/gdrive/MyDrive/Colab Notebooks/gp/model/model.hdf5 /content/gdrive/MyDrive/Colab Notebooks/gp/data/annotations/dnn.csv"

how can I train the model

**Comment by embeddedss** [comment]
Hi,I have the same problem with you. Have you solved this problem?

**Reply by antonior92** [reply]
maybe the space in `Colab Notebooks` is causing the problem?
